src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, onTabSelected, currentTab }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map((tab) => {
+          const isCurrent = tab === currentTab;
+
+          return (
+            <li
+              className={classNames({ 'is-active': isCurrent })}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => {
+                  if (currentTab !== tab) {
```

Comment:
also put this function into separate handler for better readability and maintainability

Reply:
please create a handler for this, as I said before